content/influxdb/v2.0/security/secrets/_index.md

```diff
@@ -12,29 +12,11 @@ Secrets are key-value pairs that contain sensitive information you want to contr
 access to, such as API keys, passwords, or certificates.
 There are two options for storing secrets with InfluxDB:
 
-- By default, secrets are Base64-encoded and stored in the InfluxDB embedded key value store, [BoltDB](https://github.com/boltdb/bolt).
-- You can also set up Vault to store secrets. For details, see [Store secrets in Vault](/influxdb/v2.0/security/secrets/use-vault).
+- By default, secrets are Base64-encoded and stored in the InfluxDB embedded key value store,
```

Comment:
@pierwill I'm running locally and not seeing lines 15 through 18 in doc. Did you mean to include this outside of front matter?

Reply:
I see this as the first para (edits included):
Manage secrets using any of the following:
- InfluxDB Cloud UI
- `influx`...
- InfluxDB API

Reply:
global edits for all child pages:
- Include UI as an option in the description
- include overview bullets